src/components/engagement_form_fields/subdomain.tsx

```diff
@@ -53,6 +58,10 @@ export function SubdomainFormField({
           Applications will live at:&nbsp;
           <strong>{`${getSubdomainHelperText()}`}</strong>
           <span style={{ fontStyle: 'italic' }}>{`.region.rht-labs.com`}</span>
```

Comment:
Still need this change to show the actual sub-domain choice - paired with the `region.example.com` text:
https://github.com/sdstolworthy/open-management-portal-frontend/blob/876ffd9455ba4f0ffbd0ddfdc914d6ab91df03ed/src/components/engagement_data_cards/openshift_cluster_card/openshift_cluster_card.tsx#L142-L145